containers/conformance.md

```diff
@@ -0,0 +1,220 @@
+# Zowe Conformance Criteria Related to Containerization
+
+**DRAFT**
+
+These conformance criteria are applicable for all Zowe components intent to run in containerization environment. The containerization environment could be Kubernetes or OpenShift running on Linux or Linux on Z. This may also apply to `docker-compose` running on Linux, Windows, Mac OS, or zCX.
+
+## Image
+
+In general, the image should follow [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). Below requirements are in addition to the list.
+
+### Base Image
+
+You are free to choose base image based on your requirements.
+
+For Zowe core components, here are our recommendations:
+
+- [Red Hat Universal Base Image 8 Minimal](https://developers.redhat.com/articles/ubi-faq?redirect_fragment=resources#ubi_details)
+- [Ubuntu](https://hub.docker.com/_/ubuntu)
+
+The image should be as small as possible.
```

Comment:
consider rephrasing:
The image should contain as few software packages as possible for security, and be as small as possible such as by reducing package count and layers.

Reply:
Sounds great. Will change.